src/site/content/en/blog/codelab-sign-in-form-best-practices/index.md

```diff
@@ -239,21 +235,23 @@ function togglePassword() {
 }
 ```
 
-Try out the show password logic now. View your app, enter some text into the
-password field, and then click the **Show password** button.
-Try out your new feature on multiple browsers on different operating systems.
+4. Try the show password logic now. 
+
+a. View your app. 
+
+b.Enter some text in the password field.
```

Comment:
These don't render correctly:

![image](https://user-images.githubusercontent.com/4713486/100960944-2bf42b80-34d6-11eb-870c-bdab801e9c26.png)

Please fix throughout the post by indenting like this:

```
4. Try the show password logic
   a. View your app
   b. Enter